### PR TITLE
Harmonize package deletion and claiming

### PR DESCRIPTION
**Pull request by whit537**
Update for #4305 now that #4438 is in.

- Move upsert & delete onto Package
- Make verification results easier to debug
- Smooth out a slight very wrinkle in return value for paypal_updated
- Implement and test desired behavior

### CODE REVIEW

**Comment by whit537** [comment]
Ready for review, @dowski @rohitpaulk! :)

**Comment by whit537** [comment]
Bump! I am ready to bring #4305 in for a landing!

File: gratipay/models/package/__init__.py [hunk 1]
```diff
@@ -65,14 +65,58 @@ def remote_api_url(self):
...
+
+
+    def delete(self, cursor=None):
+        """Delete the package, unlinking any team (the team itself lives on)
+        and clearing any claim.
+        """
+        cursor = cursor or self.db
+        if self.load_team(cursor):
+            self.unlink_team(cursor)
+        cursor.run("DELETE FROM claims WHERE package_id=%s", (self.id,))
```
Comment (clone1018) [inline gratipay/models/package/__init__.py]: Is the comma at the end of (self.id,) expected?

Comment (whit537) [inline gratipay/models/package/__init__.py]: Yes, it's a Python wart. We want this to be a tuple, but without the comma it is just a paranthetical expression.

**Pull request merged**